src/convertToObject.js

```diff
@@ -1,18 +1,23 @@
 'use strict';
 
 /**
- * Implement convertToObject function:
- *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
- * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrayOfStyles = sourceString.split(';')
+    .map(str => str.trim().split(':'))
+    .filter(arr => arr.length > 1);
```

Comment:
lest improve readability abit
```suggestion
  const arrayOfStyles = sourceString
    .split(';')
    .map(str => str.trim().split(':'))
    .filter(arr => arr.length > 1);
```